LibRangeCheck-3.0/LibRangeCheck-3.0.lua

```diff
@@ -207,6 +207,7 @@ tinsert(ResSpells.DRUID, 20484) -- Rebirth (40 yards, level 29)
 
 -- Hunters
 tinsert(HarmSpells.HUNTER, 75) -- Auto Shot (40 yards)
+tinsert(HarmSpells.HUNTER, 217200) -- Barbed Shot (40 yards), no for Beast Mastery
```

Comment:
The line above adds Auto Shot, which has a 40y range. I just tested that on my hunter and it has indeed a range of 40, and a range checker for range 40 works.

Can you explain what makes you want to add Barned Shot ?

Reply:
i wanted to add it cause when using the library while playing Beast Mastery "Auto Shot" doesn't exist, therefore there is no range check available in combat on Beast Mastery

Reply:
I just checked Beast Masters have Auto Shot

Reply:
hmm, okay, i received a bug report regarding the range check not working on retail (I assume the reporter refered to retail since he said the problem exists on his lvl 80 beast mastery hunter) You also checked the spell existance on retail? If so i guess maybe there is a talent or something that removes auto shot? Unfortunately i am not familar with hunter spells at all and i create the pull request based on the reporters findings

Reply:
Yes, I tested on a level 80 hunter on retail. Can you link the bug report?

Reply:
unfortunately not, it was reported in the comment section of my addon on curseforge but that post lacked information so most of the detail is inside pms on curseforge to not clutter the comment section there. But i got back to the user and asked him to verify, i will reply here once i get a reply to my pm.